oscore-cf/pom.xml

```diff
@@ -17,12 +17,6 @@
 			<groupId>org.eclipse.californium</groupId>
 			<artifactId>californium-core</artifactId>
 		</dependency>
-		<dependency>
-			<!-- requires java 1.8 -->
-			<groupId>com.augustcellars.cose</groupId>
-			<artifactId>cose-java</artifactId>
-			<version>${cose.version}</version>
```

Comment:
Please remove cose.version from the properties as well.

Reply:
Done in commit 496312141658d17078c97ca86267527ef584056f